### PR TITLE
[EGD-7922] Permanent hold cpu frequency setting window

### PR DESCRIPTION
**Pull request by PrzeBrudny**
image

### CODE REVIEW

File: module-apps/application-settings/windows/advanced/CPUModeTestWindow.cpp [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+    template <> struct enum_range<bsp::CpuFrequencyHz>
+    {
+        static constexpr int min = 0;
+        static constexpr int max = 528;
+    };
+} // namespace magic_enum
+
+namespace gui
+{
+    CPUModeTestWindow::CPUModeTestWindow(app::ApplicationCommon *app) : AppWindow(app, window::name::cpu_test_window)
```
Comment (PrzeBrudny) [inline module-apps/application-settings/windows/advanced/CPUModeTestWindow.cpp:25]: That is only Dev Advanced window -> it has no localization and proper style variables as it is not for clients. It is not attached in Release builds.

File: module-bsp/bsp/common.hpp [hunk 2]
```diff
@@ -15,13 +15,13 @@ namespace bsp
     /// Only a few thresholds are available in the current configuration
     enum class CpuFrequencyHz
```
Comment (PrzeBrudny) [inline module-bsp/bsp/common.hpp:16]: I will change name to MHz after approval (it touches a lot of files so do not want to introduce it now to clear up review).

Comment (wojtekrzepecki) [reply module-bsp/bsp/common.hpp:16]: Wouldn't it be just easier to name it like:
`
level_4MHZ,
level_12MHz,
...
` ?
Probably the values are not use anywhere

**Review by wojtekrzepecki — APPROVED** [review]

**Review by mjan-dev — APPROVED** [review]

File: module-bsp/bsp/common.hpp (hunk 2 above)
Comment (PrzeBrudny) [reply module-bsp/bsp/common.hpp:16]: Well they are used now to print in that new window :D

**Review by pawel-mudita — APPROVED** [review]

**Review by Maciej-Mudita — APPROVED** [review]

**Pull request merged**